tests/js/spec/views/installWizard.spec.jsx

```diff
@@ -80,14 +80,14 @@ describe('InstallWizard', function() {
 
     expect(
       wrapper.find('input[name="beacon.anonymous"][value="false"]').prop('checked')
-    ).toBe(true);
+    ).toBe(false);
```

Comment:
Since `beacon.anonymous.value` is `false`, shouldn't this be checked?

Reply:
Nope, see https://github.com/getsentry/sentry/blob/606c95547ce11d4a85f1279a8a699041fd31fdf8/src/sentry/static/sentry/app/views/installWizard.jsx#L67-L68

Reply:
Or is this intentional where regardless of the value, both options are always unchecked

Reply:
Yeah, it is intentional. Not sure why this test didn't fail earlier. Need to look into all this random CI stuff soon.